boards/arm64/fvp_baser_aemv8r/doc/index.rst

```diff
@@ -27,6 +27,8 @@ To Run the Fixed Virtual Platform simulation tool you must download "Armv8-R AEM
 FVP" from Arm developer [1]_ (This might require the user to register) and
 install it on your host PC.
 
+The current minimum required version of "Armv8-R AEM FVP" is 11.16.16.
```

Comment:
is it possible to check this at run-time and print an error message instead?

Reply:
Sad, I have tried several ways, no one does really work. I appreciate it if some CMake experts can tell me how to do the check.

Reply:
@tejlmand ? :)

Reply:
what you can do in CMake is something similar to this:
```cmake
execute_process(COMMAND ${ARMFVP} --version OUTPUT_VARIABLE out)
string(REPLACE "\n" ";" out ${out})

foreach(line ${out})
  if("${line}" MATCHES "Fast Models[^0-9]*([0-9][0-9]\.[0-9][0-9]\.[0-9][0-9])")
    set(armvfp_version ${CMAKE_MATCH_1})
  endif()
endforeach()

if(armvfp_version VERSION_LESS 11.16.16)
  message(WARNING "<Good warning description here>")
endif()
```
Please checkup if the match format applies to all the variants of arm VFP in use, as well as `\n` replace pattern on windows.

Reply:
@tejlmand Thank you so much. I have tried and it works! But I still have a question, this warning only prints in compile/build stage right? So, that is, after compiling/building, it cannot print check error when we use `west build -t run` with a wrong version FVP? @carlocaione do you think this will be OK?

Reply:
I've updated the PR. If the FVP version not met the requirements, it will show:
```
-- Found assembler: /home/jaxhan01/.local/zephyr-sdk/aarch64-zephyr-elf/bin/aarch64-zephyr-elf-gcc
CMake Warning at /home/jaxhan01/src/zephyrproject/zephyr/cmake/emu/armfvp.cmake:22 (message):
  Found FVP version is "11.16.16", the minimum required by the current board
  is "12.16.16".
Call Stack (most recent call first):
  /home/jaxhan01/src/zephyrproject/zephyr/CMakeLists.txt:1721 (include)
```

Reply:
> But I still have a question, this warning only prints in compile/build stage right?

@povergoing  This warning will only print during the CMake configure stage.

If you want to print an error on `west build -t run` then you will need to update the `run` target to print a warning instead of trying to execute, for example something like:
```
set(armfvp_warning_message "Found FVP version is \"${armfvp_version}\", "
      "the minimum required by the current board is \"${ARMFVP_MIN_VERSION}\".")
message(WARNING ${armfvp_warning_message})
set(ARMFVP ${CMAKE_COMMAND} -E echo ${armfvp_warning_message}
     COMMAND ${CMAKE_COMMAND} -E false
)
```

at this location: https://github.com/zephyrproject-rtos/zephyr/blob/c5cf46a6703ee805c98191f3985e253bb088f578/cmake/emu/armfvp.cmake#L21-L24

If implementing this error run-time, then please consider the handling of `ARMFVP_FLAGS `:
https://github.com/zephyrproject-rtos/zephyr/blob/c5cf46a6703ee805c98191f3985e253bb088f578/cmake/emu/armfvp.cmake#L42 


Note, a late arrival review comment, this:
https://github.com/zephyrproject-rtos/zephyr/blob/c5cf46a6703ee805c98191f3985e253bb088f578/cmake/emu/armfvp.cmake#L13
should be:
```
if (ARMFVP AND (DEFINED ARMFVP_MIN_VERSION))
```
> False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND.

Reply:
@tejlmand Really appreciate it, I will fix it in another PR.